styles/themes/teammember/_login-responsive.scss

```diff
@@ -0,0 +1,19 @@
+//== Login Page
```

Comment:
How is this being used? I can see references to content-inner class in PPM but nothing that applies this to a particular window size?